$OpenBSD$

OpenBSD patch for patch-src-Simplex-Messaging-Transport-HTTP2-Server_hs.

--- src/Simplex/Messaging/Transport/HTTP2/Server.hs.orig
+++ src/Simplex/Messaging/Transport/HTTP2/Server.hs
@@ -87,7 +87,7 @@
   tid_ <- mapM (forkIO . expireInactiveClient tls activeAt) expCfg_
   withHTTP2 bufferSize (run sniUsed tls activeAt) (clientFinished $ tlsUniq tls) tls `finally` mapM_ killThread tid_
   where
-    run sniUsed tls activeAt cfg = H.run cfg $ \req _aux sendResp -> do
+    run sniUsed tls activeAt cfg = H.run H.defaultServerConfig cfg $ \req _aux sendResp -> do
       getSystemTime >>= atomically . writeTVar activeAt
       http2Server sniUsed (tlsUniq tls) (tlsALPN tls) req (`sendResp` [])
     expireInactiveClient tls activeAt expCfg = loop
